index.html

```diff
@@ -0,0 +1,214 @@
+            <section id="video">
+                <iframe id="videoLink"  src="https://www.youtube.com/embed/7v6zCPreqYs"  allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                <div id="nameVideo">
+                    <h1>Here is why Bogibeel Bridge is special..!</h1>
+                </div>
+
+
+                <div class="publisher">
+                    <div id="userIcon" >
+                        <img src="images/userIcon.png" alt="Profil of author">
+                    </div>
+                    <div class="subscribeTab">
+                        <h1 id="nameAuthor">PIB India</h1> 
+                        <button id="subscribe">
+                            <img src="images/subscribe.PNG" alt="Click to subscribe">  
+                        </button>     
+                    </div>
+                    <div id="view">
+                        <h2>2,982,227 views</h2>
+                    </div>
+                </div>
+                
+                <div id="infoBar">
+                    <div id="likeBar">
+                        <button class="buttonAll">
+                            <img class="likeBarElement"  src="images/shareButton.png" alt="Click to share">                    
+                        </button>   
+                        <button class="buttonAll" id="addButton">
+                            <img class="likeBarElement"  src="images/addListButton.png" alt="Click to save">                
+                        </button>   
+                        <button class="buttonAll" id="threeDotButton">
+                            <img class="likeBarElement"  src="images/threeDotButton.png" alt="Click to see more options">                            
+                        </button>   
+                        <button class="buttonAll" id="likeButton">
+                            <img class="likeBarElement"  src="images/likeButton.png" alt="Click to like">
+                        </button>  
+                        <button class="buttonAll" id="unlikeButton">
+                            <img class="likeBarElement"  src="images/unlikeButton.png" alt="Click to unlike">
+                        </button>                      
+                    </div>
+                </div>
+                <br>
+                <div class="description">
+                    <h2>Published on Feb 27,2014</h2>
+                    <p>A befitting tribute to former PM #AtalBihariVajpayee, India’s longest Rail-cum-Road Bridge, #BogibeelBridge over river Brahmaputra is a construction marvel and a milestone for the development of North East. Watch to know more.</p>
+                    <h4 class="more">SHOW MORE</h4>   
+                </div>
+               
+            </section>
+            <aside id="sidebar">
+                <div id="nextBar">
+                    <h1 id="nextText">Up Next</h1>
+                    <h1 id="autoplay">Autoplay</h1>
```

Comment:
- [ ] The autoplay is a description of what the toggle button does, not a title. Avoid using h tags for elements that don't function as titles.